test/logs.js

```diff
@@ -272,12 +285,14 @@ QUnit.test( module2Test4.name, function( assert ) {
 
 	assert.deepEqual( testDoneContext, {
 		assertions: [],
-		module: module2Context.name,
-		name: module2Test3.name,
+		suiteName: module2Context.name,
+		testName: module2Test3.testName,
+		status: "skipped",
```

Comment:
Since it looks like the new event emitter API won't make it into 2.0.0, can we release the new API somewhere in a 2.x release, while keeping the existing callbacks intact? We could then remove the old callbacks in 3.0.

New events with old event data (or both old + new data) seems like a bad idea, so I think it would be better to add the new API with new data, then eventually remove the old API....